tuftool/Cargo.toml

```diff
@@ -26,7 +26,7 @@ log = "0.4"
 maplit = "1"
 olpc-cjson = { version = "0.1", path = "../olpc-cjson" }
 rayon = "1"
-reqwest = { version = "0.11", default-features = false, features = ["rustls-tls"] }
+reqwest = { version = "0.11", default-features = false, features = ["default-tls"] }
```

Comment:
Is this actually the feature you want?  For crates in the core kit, we use `rustls-tls-native-roots`.

Reply:
I think this is not the right feature flag and could result in using `native-tls` on some platforms. We don't want `tuftool` to link with openssl, even if that (indirectly) solves the problem of getting it to use the host's CA cert bundle.

Reply:
That makes sense. I have not tested with `rustls-tls-native-roots` but let me run a test.